README.md

```diff
@@ -38,13 +38,24 @@ Or use it in CSS (only the first resized image will be used, if you use multiple
 }
 ```
 
+```js
+// Outputs placeholder image as a data URI, and three images with 100, 200, and 300px widths
+const responsiveImage = require('responsive?placeholder=true&sizes[]=100,sizes[]=200,sizes[]=300!myImage.jpg');
```

Comment:
What is the syntax for using query configuration with `import`? Otherwise I'd rather keep this consistent with the rest of the examples, and create them with the Webpack v2 documentation update.